src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -23,12 +23,12 @@ internal class NTAuthentication
         private string _clientSpecifiedSpn;
 
         private int _tokenSize;
-        private Interop.SspiCli.ContextFlags _requestedContextFlags;
-        private Interop.SspiCli.ContextFlags _contextFlags;
+        private ContextFlagsPal _requestedContextFlags;
+        private ContextFlagsPal _contextFlags;
 
         private bool _isCompleted;
         private string _protocolName;
-        private SecSizes _sizes;
+        private object _sizes;
```

Comment:
yes encrypt is the only place it is referred. Can remove this property and fetched directly in Encrypt.